### ISSUE TITLE
2.3 release

### ISSUE THREAD

**Issue by tflidd** [owncloud/client#5416]
The release planning page could need an update:
https://github.com/owncloud/client/wiki/ReleaseDates

Judging from https://github.com/owncloud/client/milestones, there are already a few new features that might be worth testing with the testpilot-client. However would be great to know, when it enters beta phase and gets a feature-freeze. Especially when you manage to realize authentication via client-certificates :+1:

**Comment by guruz** [comment]
I tried to order issues in the milestone so ReadyToTest is sorted further down
https://github.com/owncloud/client/milestone/31
I also added more severities, gold tickets, assignees.

FYI The client SSL certificates branch has been merged, I'll check why the nightlies don't properly ypdate here.

I'll work to get a Change Log ready..

Still todo:
* @guruz have the Qt 5.6.1 build in the windows cross compile docker setup.
* @jturcotte improve overlay icon performance
* @SamuAlfageme try to test+close our ReadyToTest issues
* @ckamm and @ogoffart and @jturcotte finishing up current PRs, fix not-yet-working blue tickets, check current milestone.
* all: Check the open PRs and get them merged

Time plan:
We want a beta in first half of January.
Then release beginning of february

FYI @ogoffart @jturcotte @ckamm @mrow4a @felixboehm @dschmidt @SamuAlfageme
